lib/ansible/modules/network/nxos/nxos_ip_interface.py

```diff
@@ -155,9 +171,9 @@
 try:
     import ipaddress
 
-    HAS_IPADDRESS = True
+    IPADDRESS_MODULE = True
```

Comment:
Any reason you're changing this in particular? Most (all?) of the "Is this module installed" markers are named `HAS_[modulename]` for consistency

Reply:
I see; thought to have module word explicitly. HAS_SOMETHING does didn't reflect that. Anyways will revert in the interest of consistency.